planner/core/task.go

```diff
@@ -1016,47 +1016,34 @@ func CheckAggCanPushCop(sctx sessionctx.Context, aggFuncs []*aggregation.AggFunc
 	return expression.CanExprsPushDown(sc, groupByItems, client, storeType)
 }
 
+// AggPref stores the preferences of Aggregation.
```

Comment:
It is not like a `preference`, maybe `AggInfo` is more proper ?

Reply:
There's a method called `getAggInfo`, I want to distinguish with it.
`Pref` is the abbreviation for `preference`.

Reply:
Sounds like `reference` is more suitable?

Reply:
After reconsideration, I've changed to `AggInfo`.